pydoctor/test/test_astbuilder.py

```diff
@@ -1681,3 +1681,113 @@ def func():
     ''', systemcls=systemcls)
     outer = mod.contents['outer']
     assert not outer.contents
+
+@systemcls_param
+def test_constant_module(systemcls: Type[model.System]) -> None:
+    """
+    It can recognize constants define as module members.
+    """
+    mod = fromText('''
+    LANG = 'FR'
+    ''', systemcls=systemcls)
+    assert getattr(mod.contents['LANG'], 'kind') == model.DocumentableKind.CONSTANT
+    assert ast.literal_eval(getattr(mod.resolveName('LANG'), 'value') or '') == 'FR'
```

Comment:
Why `resolveName()` here instead of `contents[]`?

Reply:
No reason.